src/kili/services/export/tools.py

```diff
@@ -2,36 +2,43 @@
 import warnings
 from typing import Dict, List, Optional
 
+import requests
```

Comment:
Since the last merge on main, we do not use requests directly, but rather `http_client` which is instantiated in the `kili.client`. It can be the opportunity to remove the dependency of the export service to kili, and make it depend on http_client and graphql_client instead.

Reply:
I started doing it but it introduces a lot of changes for this PR, we should cover that during the SDK refactoring

Reply:
ok 👍  or in a different PR. That can be quick to do.